script.js

```diff
@@ -0,0 +1,46 @@
+const mobileMenu = document.createElement('div');
+function hideMobileMenu() {
+  mobileMenu.style.display = 'none';
+}
+function showMobileMenu() {
+  mobileMenu.style.display = 'block';
+}
```

Comment:
- [ ] **[REQUIRED CHANGES]**

- Please, kindly add a new line here to make your code readable and clean as now, your codebase is choked up, no newline is added to indicate the end of a function or the end of a variable